Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -641,3 +641,98 @@ public extension DepthwiseConv2D {
             padding: padding)
     }
 }
+
+/// A layer for adding Zero-padding in the temporal dimension.
+public struct ZeroPadding1D<Scalar: TensorFlowFloatingPoint>: Layer {
+    /// The padding values along the temporal dimension.
+    @noDerivative public let padding: (Int, Int)
+
+    /// Creates a Zero-padding 1D Layer.
```

Comment:
"zero-padding", not "Zero-padding", because "zero-padding" is not a proper noun. Same everywhere else.

```suggestion
    /// Creates a zero-padding 1D Layer.
```